src/components/Form/FormProvider.js

```diff
@@ -70,8 +70,12 @@ const propTypes = {
     shouldValidateOnBlur: PropTypes.bool,
 
     shouldValidateOnChange: PropTypes.bool,
+
+    shouldSetTouchedOnBlurOnly: PropTypes.bool,
```

Comment:
This prop, and the two above it, are missing comments. Can you please add them? Also, can you please change the comment for `scrollContextEnabled` prop to be all on one line?